profiles/coreos/base/package.use

```diff
@@ -39,7 +39,7 @@ net-analyzer/nmap ncat -lua
 app-admin/sudo -sendmail
 
 # avoid pulling in gnutls, disable gentoo-only bits, enable journal upload
-sys-apps/systemd -ssl curl vanilla lzma
+sys-apps/systemd -ssl curl vanilla -lz4 lzma gcrypt
```

Comment:
I noticed that if I enable both, journald compresses with lz4 by default. Fedora, because it only has lzma, uses XZ by default. So, CoreOS will be able to read Fedora journals, but not the other way around.